rfcs/0064-documentation-format.md

```diff
@@ -0,0 +1,173 @@
+---
+feature: documentation-format
+start-date: 2019-12-31
+author: Silvan Mosberger (infinisil)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+The Nix community wants to move away from using Docbook as our documentation format and replace it with something else. However it is unclear what it should be replaced with. This RFC gives a concrete process for determining the new documentation format. It does NOT say what format should be used.
```

Comment:
Should this *only* be about the format, or also the structure?

Reply:
I think changing the structure is another step and should be discussed independently.

Reply:
I think only the format for now. This RFC should be seen as an official decision that format X will be the one to use, and that people are free to work towards that without having to worry about wasting efforts.

Reply:
Isn't suitability of format closely linked to the desired structure?